app/models/user.rb

```diff
@@ -1638,7 +1643,7 @@ def number_of_flagged_posts
   end
 
   def number_of_rejected_posts
-    ReviewableQueuedPost.rejected.where(target_created_by_id: self.id).count
+    rejected_posts.length
```

Comment:
Just checking, is rejected_posts loaded yet?

If `rejected_posts` has not been loaded yet, AR fetches all records then loads it memory.`.length` returns the size of the loaded array.

Count is still better imo to be safe.